shadow-dom/focus/focus-method-delegatesFocus.html

```diff
@@ -236,5 +236,52 @@
   assert_equals(shadowRoot.activeElement, belowSlots);
 }, "focus() on host with delegatesFocus and already-focused non-first shadow descendant");
 
+function createNestedHosts(innerDelegatesFocus) {
+  // Structure:
+  // <div> outerHost
+  //   <input> outerLightChild
+  //   #shadowRoot outerShadow delegatesFocus=true
+  //     <span> innerHost
+  //       #shadowRoot inneShadow delegatesFocus=true/false
+  //         <input> innerShadowChild
+  //     <input> outerShadowChild
+  const outerHost = document.createElement('div');
+  const outerLightChild = document.createElement('input');
+  outerHost.appendChild(outerLightChild);
+  const innerHost = document.createElement('span');
+  const outerShadow = outerHost.attachShadow({mode: 'closed', delegatesFocus:true});
+  outerShadow.appendChild(innerHost);
+  const outerShadowChild = document.createElement('input');
+  outerShadow.appendChild(outerShadowChild);
+
+  const innerShadow = innerHost.attachShadow({mode: 'closed', delegatesFocus:innerDelegatesFocus});
+  const innerShadowChild = document.createElement('input');
+  innerShadow.appendChild(innerShadowChild);
+
```

Comment:
You need to set tabIndex=0 on innerHost and outerHost.

Reply:
We don't need tabindex attributes though we may add them.
A shadow host with delegatesFocus can't be a focusable area regardless of its tabindex attribute value.

Reply:
Ah, I misread the spec. Makes sense. It's a bit odd that an element becomes "focusable" when `delegatesFocus` is present but it's not the worst thing in the world.